code/modules/mob/living/carbon/human/human.dm

```diff
@@ -679,6 +679,28 @@
 		else
 			return ""
 
+/mob/living/carbon/human/Hear(message, atom/movable/speaker, var/datum/language/speaking, raw_message, radio_freq)

```

Comment:
Yeah, it's from Living/Hear

The behavior I want is so deeply nested in there that I can't think of a way to do it other than...
A) Cast src as a human, which is badbadbad
B) This
C) Make yet another separate proc inside mob, but this time make it earwearingheadsetmuffs() and have it return 0, and ONLY override it here in human with a test. I'd rather not crowd mob with a item-time proc only used by humans though.

This seems like the most pretty way to do it.